packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.service.js

```diff
@@ -21,4 +21,21 @@ export default class {
       },
     );
   }
+
+  getDatastoreLocation(serviceName, datacenterId, filerId) {
+    if (!datacenterId) {
+      return this.OvhHttp.get(
+        `/dedicatedCloud/${serviceName}/filer/${filerId}/location`,
+        {
+          rootPath: 'apiv6',
+        },
+      );
+    }
+    return this.OvhHttp.get(
```

Comment:
use $http instead

Reply:
Didn't know that, was keeping the same coding style as other files, switching to $http

Reply:
No worries, we try to remove or code with no legacy style (all native).

Take care, no need to use rootPath with $http now I believe

Reply:
Yup, I just tried, works without rootPath, removing it now